src/test/java/com/teragrep/pth10/logicalOperationTest.java

```diff
@@ -260,229 +324,314 @@ public void parseWithQuotesInsideQuotesCatalystTest() {
         });
     }
 
-    @Disabled
-    @Test // disabled on 2022-05-16 TODO Convert to dataframe test
-    public void parseAnd1Test() throws AnalysisException {
-        String q, e, result;
-        q = "index=kafka_topic a1 a2";
```

Comment:
test without parenthesis?

Reply:
I think this one tests without parenthesis as well  **testUnquotedSearchString()**